src/pudl/metadata/sources.py

```diff
@@ -268,7 +268,7 @@
         ),
         "source_file_dict": {
             "respondents": "Coal and high-sulfur fueled plants over 25MW",
-            "records_liberated": "~800 million",
+            "records_liberated": "924 million",
```

Comment:
I suspect we're going to be inconsistent about updating this each quarter, unless we add it to our "add new data" issue template task list - suggest either rounding to ~900 million or ~1 billion here or adding it to the template to update with regularity.

Reply:
Rounded to order of magnitude. Would be nice to generate these things automatically.

Reply:
Agreed, definitely out of scope but would be nice.